index.html

```diff
@@ -7,6 +7,6 @@
   <title>Document</title>
 </head>
 <body>
-  
+  <h1>hi there</h1>
```

Comment:
add more

Reply:
https://dev.to/colocodes/learn-how-to-use-git-and-github-in-a-team-like-a-pro-2dk7